app/src/main/java/ru/yandex/praktikumchatapp/MainActivity.kt

```diff
@@ -71,7 +71,7 @@ fun ChatScreen(
     modifier: Modifier = Modifier
 ) {
     val viewModel = remember { ChatViewModel() }
-    val messagesList = viewModel.messages.observeAsState(emptyList())
+    val messagesList = viewModel.messages.collectAsStateWithLifecycle()
```

Comment:
Можно пойти дальше и воспользоваться проперти делегатом:
```suggestion
    val messagesList by viewModel.messages.collectAsStateWithLifecycle()
```
Это позволит обращаться к `messagesList` напрямую, а не используя свойство `value`